packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts

```diff
@@ -47,12 +46,12 @@ let totalSupportThresholdPct = '500';
 let census = '1000';
 let actions = createDummyActions(DAO_TOKEN_ADDRESS, '0', '0x00000000');
 
-test('Run Allowlist Voting (handleVoteCreated) mappings with mock event', () => {
+test('Run Addresslist Voting (handleVoteCreated) mappings with mock event', () => {
   // create state
-  let erc20VotingPackage = new AllowlistPackage(
+  let erc20VotingPlugin = new AddresslistPlugin(
```

Comment:
rename `erc20VotingPlugin` to `addresslistPlugin`

Reply:
Amended